src/styles/page-template/index.md.njk

```diff
@@ -41,7 +41,7 @@ You can customise the page template, for example, by adding a service name and n
 
 ### Configuring Nunjucks
 
-When using Nunjucks you’ll need to add `node_modules/govuk-frontend/govuk/` and `node_modules/govuk-frontend/govuk/components/`
+When using Nunjucks you’ll need to add `node_modules/govuk-frontend/` and `node_modules/govuk-frontend/govuk/components/`
```

Comment:
I think our suggested approach is not to add components into the paths, and do this instead?

```
{% from "govuk/components/foo/macro.njk" import govukFoo %}
```

```suggestion
When using Nunjucks you’ll need to add `node_modules/govuk-frontend/`
```

Reply:
Yeah this is correct, the example below will need updating too.

Reply:
Makes sense 👍